backend/account/tests/test_query.py

```diff
@@ -54,6 +54,85 @@ def test_invalid_update_account(self):
             query.update_user({"first_name": "Rob"})
 
 
+class PrivacyTests(TestCase):
+    def test_create_privacies(self):
+        user = UserFactory()
+        priv = query.create_privacies(user.id)
+        self.assertEqual(models.Privacy.objects.count(), 1)
+        self.assertEqual(priv.user.id, user.id)
+
+    def test_get_privacies(self):
+        user = UserFactory()
+        priv = query.create_privacies(user.id)
+        privFetch = query.get_privacies(user.id)
+        self.assertEqual(priv.id, privFetch.id)
+
+    def test_invalid_privacies_get(self):
+        user = UserFactory()
+        query.create_privacies(user.id)
+        self.assertEqual(models.Privacy.objects.count(), 1)
+        # test invalid getting
+        with self.assertRaises(ObjectDoesNotExist):
+            query.get_privacies(user.id + 1)
+
+
+class UpdatePrivacyTests(TestCase):
```

Comment:
Same idea in these `update` tests, use `PrivacyFactory` rather than`create_privacies`. Also, use `Privacy.objects.get()` rather than `get_privacies`.

Reply:
Why would I not use `get_privacies` when I am testing its functionality?

Reply:
You're testing the `update` functionality in this class right? So you'll want to isolate the tests to not rely on the functionality of other (potentially faulty) methods

Reply:
My bad, the silly Git preview includes line 76 with your comment, which should be using `get_privacies`. Noted.

Reply:
👀